lxd/images.go

```diff
@@ -99,6 +99,13 @@ func unpack(file string, path string) error {
 	} else if strings.HasPrefix(extension, ".squashfs") {
 		command = "unsquashfs"
 		args = append(args, "-f", "-d", path, "-n")
+
+		mem, err := deviceTotalMemory()
+		mem = mem / 1024 / 1024 / 10
```

Comment:
This is non-intuitive for me; even with the commit message it took me a bit to work through the logic.  I think it's especially confusing for people used to power-of-two math, the `10` seems so out of place.  Any objection to changing the code to something like this?  This assumes the comparison against 2.5gb instead of 2gb was unintentional.

```go
if err == nil && mem <= (1<<31) {
    mem = mem / 1024 / 1024 / 10
    ...
}
```